openmrs/apps/reports/reports.json

```diff
@@ -428,6 +428,48 @@
                         "sqlPath": "/var/www/bahmni_config/openmrs/apps/reports/TESTING_REPORT/sql/indicator6d_not_available.sql"
                     }
                 },
+                {
```

Comment:
@tmvumbi2 I think from now on, we can name the sql files with their final name instead of not_available. I don't think we gain much from doing it the old way, and we would save a lot of time when we implement the indicator. What do you think?

Reply:
yes I agree :+1